src/main/java/io/cryostat/net/web/http/api/v2/HttpApiV2Module.java

```diff
@@ -184,4 +184,24 @@ abstract RequestHandler bindTargetCredentialsDeleteHandler(
     @Binds
     @IntoSet
     abstract RequestHandler bindReportGetHandler(ReportGetHandler handler);
+
+    @Binds
+    @IntoSet
+    abstract RequestHandler bindRecordingMetadataPatchHandler(
+            RecordingMetadataPatchHandler handler);
+
+    @Binds
+    @IntoSet
+    abstract RequestHandler bindTargetRecordingMetadataPatchHandler(
+            TargetRecordingMetadataPatchHandler handler);
+
+    @Binds
+    @IntoSet
+    abstract RequestHandler bindRecordingMetadataPatchBodyHandler(
+            RecordingMetadataPatchBodyHandler handler);
+
+    @Binds
+    @IntoSet
+    abstract RequestHandler bindTargetRecordingMetadataPatchBodyHandler(
+            TargetRecordingMetadataPatchBodyHandler handler);
```

Comment:
Could you move these new handlers from the V2 module and API v2.1 namespace into the Beta module? I think we might want to hold off on including this in 2.1.0 since that's coming up very soon and this feature isn't in the current release planning roadmap. If we keep it in Beta then we're free to make more changes to it before the next major release where we promote it to the main API.